oscovida/covid19dh.py

```diff
@@ -0,0 +1,341 @@
+import datetime
```

Comment:
My bad, that should have been there at the start.

This file is pretty much [the covid19dh python interface](https://github.com/covid19datahub/Python/tree/master/covid19dh) with a few changes and improvements, like caching to disk.

I left the file name as just `covid19dh` to make the implementation easier. Previously `covid19dh` was a dependency, and then in the other files I'd just do `import covid19dh` and use the module. When I decided to make some changes to it, I put everything into the same file so that `from . import covid19dh` would work as a drop-in replacement.

The plan is to, later on, do a PR for the upstream version and merge these changes into it, if they're approved then this file can be deleted and I'll just add it back in as a dependency.